### PR TITLE
fix(openapi): support nullish consts

### PR DESCRIPTION
**Pull request by WoH**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test/spec/openapi/schema.js [hunk 1]
```diff
@@ -655,7 +655,9 @@ test('support "const" keyword', async t => {
           obj: {
             type: 'object',
             properties: {
-              constantProp: { const: 'my-const' }
+              constantProp: { const: 'my-const' },
```
Comment (Uzlopak) [inline test/spec/openapi/schema.js:658]: please also add a test for false and empty string.

**Review by Uzlopak — APPROVED** [review]
seems right, but I would like to have a second opinion by @ivan-tymoshenko

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by WoH** [comment]
> I'm not really deep into this library. The only thing I would check (add a test) is how it would work with `nullable` keyword and type `null`. For example:
> 
> ```json
> {
>   "type": "string",
>   "nullable": false,
>   "const": null
> }
> ```
> 
> and
> 
> ```json
> {
>   "type": "string",
>   "const": null
> }
> ```
> 
> Both these schemas are invalid (there is no value that would match them). You want to be sure that you handle cases like that correctly.

In both of these cases I'd argue it's not the job of this library to figure out what may have been implied, but just convert it to an equally useless (albeit OAS compatible) schema.

I would like to mention that one could make an argument for converting:
```json
 {
  "const": null
}
```

to:

```json5
 {
  "enum": [null],
  "nullable": true // with this added
}
```

but that is is not required from my understanding of the spec (which seems to be in line with the swagger-validator).

**Review by ivan-tymoshenko — APPROVED** [review]
